src/Munee/Asset/Type/Css.php

```diff
@@ -204,4 +205,42 @@ protected function fixRelativeImagePaths($content, $originalFile)
 
         return $content;
     }
+
+    private static function parseImports($content,$origFile)
```

Comment:
Please use `protected`, also put a space after the comma in the params.  This goes for any params and comma delimited lists.